bindings/java/c/evmc-vm.c

```diff
@@ -90,12 +90,12 @@ JNIEXPORT void JNICALL Java_org_ethereum_evmc_EvmcVm_execute(JNIEnv* jenv,
                                                              jint jrev,
                                                              jobject jmsg,
                                                              jobject jcode,
-                                                             jint jcode_size,
                                                              jobject jresult)
 {
     (void)jcls;
     struct evmc_message* msg = (struct evmc_message*)(*jenv)->GetDirectBufferAddress(jenv, jmsg);
     assert(msg != NULL);
+    size_t code_size = (size_t)(*jenv)->GetDirectBufferCapacity(jenv, jcode);
```

Comment:
Add assert that `code_size` is not negative. Probably should be done before casting to `size_t`.

Reply:
Done.